site/components/Navigation/NavItemDesktop.tsx

```diff
@@ -1,24 +1,48 @@
 import React, { FC } from "react";
 import * as styles from "./styles";
 import Link from "next/link";
+import { cx } from "@emotion/css";
 
 interface Props extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
   name: string;
-  url: string;
+  url?: string;
+  subMenu?: JSX.Element[];
   active?: boolean;
 }
 
-export const NavItemDesktop: FC<Props> = ({ active, name, url, ...props }) => {
+export const NavItemDesktop: FC<Props> = ({
+  active,
+  name,
+  url,
+  subMenu,
+  ...props
+}) => {
   return (
-    <Link href={url}>
-      <a
-        {...props}
-        className={styles.navMain_DesktopLink}
-        href={url}
-        data-active={active?.toString()}
-      >
-        {name}
-      </a>
-    </Link>
+    <div className={cx(styles.navMain_DesktopSelectContainer)}>
+      {url ? (
+        <Link href={url}>
+          <a
+            className={cx(styles.navMain_DesktopLink, {
+              dropdown: subMenu !== undefined,
+            })}
+            href={url}
```

Comment:
we shouldn't pass the href here because `next/link` already passes it down with pre-fetching and locale magic.

Reply:
removed